src/components/destination_list.js

```diff
@@ -20,12 +20,18 @@ const DestinationList = ({
   unselectItems,
   selectedItemRenderer,
   noItemsRenderer,
-  withGrouping
+  withGrouping,
+  enableRightDefault,
+  items
 }) => {
   const SelectionStatusRenderer = selectionStatusRenderer;
   const updatedSelectedItems = withGrouping
     ? groupItems(selectedItems)
     : selectedItems;
+
+  const defaultColumns = enableRightDefault
+    ? items.filter(item => item.disabled && item)
```

Comment:
Why is disabled what's deciding if something is default or not? Seems counter intuitive.